### PR TITLE
Refactor and various fixes

### PR DESCRIPTION
**Pull request by topless**

### CODE REVIEW

**Comment by topless** [comment]
This is expected to fail since it cannot pul the `npm` package.

**Comment by ntarocco** [comment]
> This is expected to fail since it cannot pul the `npm` package.

Ok then let's wait for the npm to be ready!

**Comment by topless** [comment]
- Replaced all cominations of invenio-record-editor to invenio-records-editor
- The project now points to inveniosoftware repository
- Replaced all author entries with inveniosoftware
- Fixes for testing when cannot find bundles

File: invenio_records_editor/views.py [hunk 1]
```diff
@@ -17,9 +18,9 @@
 def create_editor_blueprint(app):
```
Comment (ntarocco) [inline invenio_records_editor/views.py:18]: Question, why this function?
We can just register the Blueprint in the entrypoints, without having to add the registration manually when creating the app.

File: RELEASE-NOTES.rst [hunk 2]
```diff
@@ -19,18 +19,18 @@ What's new
...
+   $ pip install invenio-records-editor==0.1.0
 
 Documentation
 -------------
 
-   https://invenio-record-editor.readthedocs.io/
+   https://invenio-records-editor.readthedocs.io/
 
-Happy hacking and thanks for flying Invenio-Record-Editor.
+Happy hacking and thanks for flying Invenio-Records-Editor.
```
Comment (ntarocco) [inline RELEASE-NOTES.rst:29]: LoL, even if I personally like it, not sure we want to keep it ;)

**Review by ntarocco — COMMENTED** [review]
Would it be possible to have the mock data in the example app and not in the main `src`?

**Pull request merged**